class/defaults.yml

```diff
@@ -2,3 +2,105 @@ parameters:
   topolvm:
     =_metadata: {}
     namespace: syn-topolvm
+
+    images:
+      topolvm:
+        registry: quay.io
+        repository: topolvm/topolvm-with-sidecar
+        tag: 0.8.3
+      csi_provisioner:
+        registry: k8s.gcr.io
+        repository: sig-storage/csi-provisioner
+        tag: v2.2.1
+
+    deviceclasses:
+      ssd:
+        volumegroup: vgssd
+        sparegb: 10
+        default: false
+
+    storageclasses:
+      ssd-local:
+        class: ssd
+        fstype: ext4
+        isdefault: false
```

Comment:
not required when using the storageclass component library. Select default class via `parameters.storageclass.defaultClass`.